src/applications/disability-benefits/686/config/form.js

```diff
@@ -89,17 +166,142 @@ function createSpouseLabelSelector(nameTemplate) {
     };
   });
 }
-function calculateChildAge(form, index) {
-  if (form.dependents[index].childDateOfBirth) {
-    const childAge = form.dependents[index].childDateOfBirth;
-    return moment().diff(childAge, 'years');
-  }
-  return null;
+
+// given a `key` string that optionally includes the substring `[INDEX]`,
+// replace the instance of `[INDEX]` with the passed-in `index` value and return
+// the new key
+//
+// ex: given 'marriages[INDEX].locationOfMarriage' and `0` return
+// 'marriages[0].locationOfMarriage'
+function insertRealIndexInKey(key, index) {
+  return key.replace(/(?<=\[)INDEX(?=\])/, index);
```

Comment:
grr...

I think I'd prefer restoring to what I had previously since it's safer than just looking for INDEX. And I'm totally worried about an issue that'll never come up, I know.